src/lib/Hydra/Controller/Build.pm

```diff
@@ -256,7 +253,12 @@ sub serveFile {
             $type = $info->{mime_with_encoding};
         }
         $c->response->content_type($type);
-        $c->forward('Hydra::View::Plain');
+
+        $c->res->finalize;
+        my @cmd = ("nix", "--experimental-features", "nix-command", "store", "cat", "--store", getStoreUri(), "$path");
+        my $ok = run \@cmd, '>', $c->res->output;
```

Comment:
Does this actually work?

The unit test suite says otherwise:

```
| 0E0FFC9A-93E8-11F0-8F8 | Hydra/Controller/Build | Test downloading build |
| 8-B17F2BFB4156         | /download.t            |  products (regression  |
```